atomic_reactor/util.py

```diff
@@ -1269,6 +1270,18 @@ def read_yaml_from_file_path(file_path, schema):
     return read_yaml(yaml_data, schema)
 
 
+def read_yaml_from_url(url, schema):
+    session = get_retrying_requests_session()
+    resp = session.get(url, stream=True)
+
+    f = io.BytesIO()
+    for chunk in resp.iter_content(chunk_size=DEFAULT_DOWNLOAD_BLOCK_SIZE):
+        f.write(chunk)
+
+    f.seek(0)
+    return read_yaml(f.read().decode(), schema)
```

Comment:
To be consistent py2/3 shouldn't we enforce decode to use utf8?

Reply:
Huh, utf8 is not the default on python2? TIL